src/targets/chrome/get-selector-box-size.js

```diff
@@ -1,10 +1,42 @@
 const getSelectorBoxSize = (window, selector) => {
-  const element = window.document.querySelector(selector);
-  if (!element) {
-    throw new Error('Unable to find element');
+  const elements = [...window.document.querySelectorAll(selector)];
+
+  if (elements.length === 0) {
+    throw new Error('Unable to find selector');
```

Comment:
Let's keep the error message consistent in case some other part of the codebase looks for it.

Reply:
What do you mean by consistent? 😕

Reply:
We need to make sure it's the same message being tested for in [`create-chrome-target.js`](https://github.com/oblador/loki/blob/a455ef88929bba30fa196e40f8a76975bfeb9150/src/targets/chrome/create-chrome-target.js#L149)

Reply:
Done! ✅